src/include/iscsi-lio-server/widgets.rb

```diff
@@ -1445,8 +1448,24 @@ def handleClient(key, event)
 
     def validateClient(key, event)
       event = deep_copy(event)
-      ret = true
-      ret
+      clients = Convert.convert(UI.QueryWidget(:clnt_table, :Items),
```

Comment:
convert is useless in new code, so it can be simple `clients = UI.QueryWidget(:clnt_table, :Items)`